keycodes.go

```diff
@@ -0,0 +1,261 @@
+		"KeyCoffee":           152,
+		"KeyDirection":        153,
+		"KeyCyclewindows":     154,
+		"KeyMail":             155,
+		"KeyBookmarks":        156,
+		"KeyComputer":         157,
+		"KeyBack":             158,
+		"KeyForward":          159,
+		"KeyClosecd":          160,
+		"KeyEjectcd":          161,
+		"KeyEjectclosecd":     162,
+		"KeyNextsong":         163,
+		"KeyPlaypause":        164,
+		"KeyPrevioussong":     165,
+		"KeyStopcd":           166,
+		"KeyRecord":           167,
+		"KeyRewind":           168,
+		"KeyPhone":            169,
+		"KeyIso":              170,
+		"KeyConfig":           171,
+		"KeyHomepage":         172,
+		"KeyRefresh":          173,
+		"KeyExit":             174,
+		"KeyMove":             175,
+		"KeyEdit":             176,
+		"KeyScrollup":         177,
+		"KeyScrolldown":       178,
+		"KeyKpleftparen":      179,
+		"KeyKprightparen":     180,
+		"KeyNew":              181,
+		"KeyRedo":             182,
+		"KeyF13":              183,
+		"KeyF14":              184,
+		"KeyF15":              185,
+		"KeyF16":              186,
+		"KeyF17":              187,
+		"KeyF18":              188,
+		"KeyF19":              189,
+		"KeyF20":              190,
+		"KeyF21":              191,
+		"KeyF22":              192,
+		"KeyF23":              193,
+		"KeyF24":              194,
+		"KeyPlaycd":           200,
+		"KeyPausecd":          201,
+		"KeyProg3":            202,
+		"KeyProg4":            203,
+		"KeyDashboard":        204,
+		"KeySuspend":          205,
+		"KeyClose":            206,
+		"KeyPlay":             207,
+		"KeyFastforward":      208,
+		"KeyBassboost":        209,
+		"KeyPrint":            210,
+		"KeyHp":               211,
+		"KeyCamera":           212,
+		"KeySound":            213,
+		"KeyQuestion":         214,
+		"KeyEmail":            215,
+		"KeyChat":             216,
+		"KeySearch":           217,
+		"KeyConnect":          218,
+		"KeyFinance":          219,
+		"KeySport":            220,
+		"KeyShop":             221,
+		"KeyAlterase":         222,
+		"KeyCancel":           223,
+		"KeyBrightnessdown":   224,
+		"KeyBrightnessup":     225,
+		"KeyMedia":            226,
+		"KeySwitchvideomode":  227,
+		"KeyKbdillumtoggle":   228,
+		"KeyKbdillumdown":     229,
+		"KeyKbdillumup":       230,
+		"KeySend":             231,
+		"KeyReply":            232,
+		"KeyForwardmail":      233,
+		"KeySave":             234,
+		"KeyDocuments":        235,
+		"KeyBattery":          236,
+		"KeyBluetooth":        237,
+		"KeyWlan":             238,
+		"KeyUwb":              239,
+		"KeyUnknown":          240,
+		"KeyVideoNext":        241,
+		"KeyVideoPrev":        242,
+		"KeyBrightnessCycle":  243,
+		"KeyBrightnessZero":   244,
+		"KeyDisplayOff":       245,
+		"KeyWimax":            246,
+		"KeyRfkill":           247,
+		"KeyMicmute":          248,
+		"keyMax":              248,
+	}
+
+	for k, v := range kc {
+		keycode = strings.ReplaceAll(keycode, k, strconv.Itoa(v))
```

Comment:
We should call `strings.ToLower` on both `keycode` and `k` here, so we match any capitalization of the string.